tests/reset_workflow.go

```diff
@@ -651,6 +652,113 @@ func (s *FunctionalSuite) testResetWorkflowReapplyBuffer(
 
 }
 
+func (s *FunctionalSuite) TestUpdateMessageInLastWFT() {
+	s.dynamicConfigOverrides = map[dynamicconfig.Key]interface{}{
+		dynamicconfig.FrontendEnableUpdateWorkflowExecutionAsyncAccepted: true,
+	}
+	tv := testvars.New(s.T().Name())
+	workflowId := "wfid"
+	workflowTypeName := "wftype"
+	taskQueueName := "tq"
+	identity := "worker1"
+	workflowType := &commonpb.WorkflowType{Name: workflowTypeName}
+	taskQueue := &taskqueuepb.TaskQueue{Name: taskQueueName, Kind: enumspb.TASK_QUEUE_KIND_NORMAL}
+	updateId := tv.UpdateID("update-id")
+
+	we, err := s.engine.StartWorkflowExecution(NewContext(), &workflowservice.StartWorkflowExecutionRequest{
+		RequestId:           uuid.New(),
+		Namespace:           s.namespace,
+		WorkflowId:          workflowId,
+		WorkflowType:        workflowType,
+		TaskQueue:           taskQueue,
+		Input:               nil,
+		WorkflowRunTimeout:  durationpb.New(100 * time.Second),
+		WorkflowTaskTimeout: durationpb.New(1 * time.Second),
```

Comment:
For "long" timeouts there is `tv.InfinityTimeout()` - which means "I don't care, I just want it to be long enough". And if you are not depends on WFT timeout, you shouldn't specify it too.